fe/fe-core/src/main/java/com/starrocks/catalog/CatalogRecycleBin.java

```diff
@@ -155,50 +158,76 @@ public synchronized Database getDatabase(long dbId) {
         return null;
     }
 
-    public synchronized Table recycleTable(long dbId, Table table) {
+    public synchronized void recycleTable(long dbId, Table table, boolean recoverable) {
```

Comment:
If you change parameter recoverTable into recycleTime, it will be much more easy to understand